instagram-app/src/App.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+//import logo from './logo.svg';
+import './App.css';
+import dummyData from './dummy-data';
```

Comment:
import would be correct here if dummy-data was currently on the same level. As is, it is not and would have to be imported from outside the src folder...which doesn't work with React.